### DISCUSSION TITLE
PCManFM unable to access thumb drive from OS installed to external SSD

### DISCUSSION THREAD

**Discussion by edwardp3** [lxqt/lxqt#2417] [Q&A]
I have Fedora installed on an external SSD (USB-connected). In trying to import a bookmarks file (HTML) from a separate USB thumb drive into the browser, PCManFM is not recognizing the thumb drive. The Removable Media applet in the Panel, sees the device. If I click its eject icon, it displays a message that _unmounting failed - remote peer disconnected_. But the device then appears in PCManFM under Devices, along with the SSD itself. If I then click the thumb drive entry under Devices, PCManFM displays an error window with _Remote peer disconnected_ and the device disappears from under Devices. In addition, it is seen with the `lsblk` command, but if I try to mount the thumb drive from a terminal window, it tells me it's not found in `/etc/fstab`. It does not matter whether the thumb drive is already in a USB port at bootup, or if inserted after the fact.

Is what I am trying to do, not possible?

Please note that on the desktop's internal HDD (with Debian), PCManFM sees the thumb drive perfectly and it is accessible.

Thanks in advance.

**Comment by stefonarch** [comment]
No idea what is going wrong here but
> In addition, it is seen with the lsblk command, but if I try to mount the thumb drive from a terminal window, it tells me it's not found in /etc/fstab

When you get this error about fstab your command was probably wrong, it should be something like

`sudo mount /dev/sdXY /mnt` with XY=identifier of the partion, e.g. sd**b4**

**Reply by edwardp3** [reply]
Plugged in the thumb drive, LXQt immediately displayed _Remote peer disconnected_ again.

Tried the above command (was sdd1), prompted for password, no output. I then tried sudo mount /dev/sdd and /dev/sdd1, both responded: `can't find in /etc/fstab`.

```
~]$ sudo mount /dev/sdd/
mount: /dev/sdd/: can't find in /etc/fstab.
```

```
~]$ sudo mount /dev/sdd1/
mount: /dev/sdd1/: can't find in /etc/fstab.
```

PCManFM is now not seeing the thumb drive.

If I try accessing it from the command line, I get Permission Denied. The su and sudo commands do not work in this instance. Account is in the wheel (admin) group.

```
~]$ sudo /dev/sdd1
sudo: /dev/sdd1: command not found
```


```
~]$ /dev/sdd1
bash: /dev/sdd1: Permission denied

```

**Reply by edwardp3** [reply]
With Debian installed to the internal HDD on the same system, if I insert the same thumb drive, I get a _Removable medium is inserted_ window and the option is to _Open in file manager_. 

With Fedora, I don't see any of this.

**Comment by stefonarch** [comment]
Re-read my comment please. Your mount command is  wrong, no mount point provided. And probably there is another issue around, beside this-

**Reply by edwardp3** [reply]
```
sdd      8:48   1 117.4G  0 disk 
└─sdd1   8:49   1 117.4G  0 part 
```


` ~]$ sudo mount /dev/sdd1 /mnt`

Still no response to command. Assuming it's mounted, still cannot access it. Not listed under Devices in PCManFM.

![pcmanfm](https://user-images.githubusercontent.com/73129450/235361405-ea60f754-3cd0-414c-943b-c3c20e1b119f.png)

**Comment by stefonarch** [comment]
> Still no response to command

No respond means executed without errors. Open the directory. 
Try `ls /mnt` or `pcmanfm-qt  /mnt`

**Reply by edwardp3** [reply]
/mnt is empty:

![mnt](https://user-images.githubusercontent.com/73129450/235366905-5e4e57e8-da2d-4f2f-99fa-6e8763b0579b.png)


As soon as I plug the thumb drive in, I always get this:

![pluginusbthumbdrive](https://user-images.githubusercontent.com/73129450/235366864-1d7b7d0e-72d8-4dd8-b97f-7aa73c546b22.png)

**Comment by tsujan** [comment]
@edwardp3
Since you can't mount it in a terminal, your problem should be independent of LXQt.

**Reply by edwardp3** [reply]
Hi @tsujan 

Thank you for commenting. Somewhere in the middle of finding out what's going on, the contents of the thumb drive in question became erased. I then used Gparted to reformat it with FAT32 using Debian and was able to recover mostly everything (so far) from the other thumb drive (read-only, ext4) and the internal HDD. 

The FAT32 thumb drive was then successfully detected by Fedora and PCManFM-Qt and it is also read/write. I was then able to import the bookmarks file into the browser on Fedora.

_How the contents became erased, I haven't a clue._

@stefonarch Thank you for the above assistance.
